include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
     
     /* Extensions Exposed as Features: */
```

Comment:
its better to list them in "Registered Extension Number" rather than alphabetical

Reply:
that's what vulkan.gpuinfo offered, also it's really hard for me to reorder them now, I  already went through them multiple times to remove useless stuff

Reply:
its not very hard for me to verify them all

Reply:
missing from the listing (will keep on updating)
```
VK_KHR_sampler_mirror_clamp_to_edge
```

Reply:
I removed every NVX because you said it's experimental stuff
I removed VK_KHR_sampler_mirror_clamp_to_edge because we already handles that in features/limits.
I removed duplicated stuff we have above in features or limits

Reply:
> I removed every NVX because you said it's experimental stuff
These are the only two exceptions to the "no NVX"  rule, these extensions will never be official cause they're for executing CUDA kernels in Vulkan :rofl:

> I removed VK_KHR_sampler_mirror_clamp_to_edge because we already handles that in features/limits.

ah ok I forgot thats already handled

Reply:
Stuff that's not in your list, but is not good to expose anyway
```
VK_HUAWEI_invocation_mask // its dumb to compact raygen this way
VK_HUAWEI_subpass_shading // meh, crossplatform VRS is better
VK_FUCHSIA_... // its not a real OS
```

Reply:
would be useful to list `VK_KHR_spirv_1_4` just to say we dont expose it because we always enable it